app/src/main/BUILD

```diff
@@ -32,7 +32,6 @@ kt_android_library(
         artifact("androidx.lifecycle:lifecycle-viewmodel"),
         artifact("androidx.lifecycle:lifecycle-common"),
         artifact("androidx.drawerlayout:drawerlayout"),
-        artifact("org.jetbrains:annotations"),
```

Comment:
@mig35  why do we remove this here? I assume latest release includes this for us maybe?

Reply:
yep, this is definitely true. Kotlin rules now has it installed